docs/content/concepts/index.md

```diff
@@ -1,116 +1,3 @@
-object and potentially the schema the user sees.
-
-## Index (e.g. Workspace Index)
-
-An index is the authoritative list of a particular API in their source of truth across the system.  For instance, in
-order for a user to see all the workspaces they have available, they must consult the workspace index to return a list
-of their workspaces.  It is expected that indices are suitable for consistent LIST/WATCHing (in the kubernetes sense) so
-that integrations can be built to view the list of those objects.
-
-Index in the control plane sense should not be confused with secondary indices (in the database sense), which may be
-used to enable a particular index.
-
-## Shard
-
-A failure domain within the larger control plane service that cuts across the primary functionality. Most distributed
-systems must separate functionality across shards to mitigate failures, and typically users interact with shards through
-some transparent serving infrastructure.  Since the primary problem of building distributed systems is reasoning about
-failure domains and dependencies across them, it is critical to allow operators to effectively match shards, understand
-dependencies, and bring them together.
-
-A control plane should be shardable in a way that maximizes application SLO - gives users a tool that allows them to
-better define their applications not to fail.
-
-## API Binding
-
-The act of associating a set of APIs with a given logical cluster.  The Workspace model defines one particular
-implementation of the lifecycle of a logical cluster and the APIs within it.  Because APIs and the implementations that
-back an API evolve over time, it is important that the binding be introspectable and orchestrate-able - that a consumer
-can provide a rolling deployment of a new API or new implementation across hundreds or thousands of workspaces.
-
-There are likely a few objects involved in defining the APIs exposed within a workspace, but in general they probably
-define a spec (which APIs / implementations to associate with) and a status (the chosen APIs / implementations that are
-currently bound), allow a user to bulk associate APIs (i.e. multiple APIs at the same time, like “all knative serving
-APIs”), and may be defaulted based on some attributes of a workspace type (all workspaces of this “type” get the default
-Kube APIs, this other “type” get the knative apis).
-
-The evolution of an API within a workspace and across workspaces is of key importance.
-
-## Syncer
-
-A syncer is installed on a SyncTarget and is responsible for synchronizing data between kcp and that cluster.
-
-## Location
-
-A collection of SyncTargets that describe runtime characteristics that allow placement of applications.
-Characteristics are not limited but could describe things like GPU, supported storage, compliance or
-regulatory fulfillment, or geographical placement.
+{% include "partials/section-overview.html" %}
```

Comment:
Is the partial used because Mkdocs can't do loops in md files?

Reply:
It's shared among all the section overview pages